django/website/custom_organizations/templates/organizations/organization_form.html

```diff
@@ -0,0 +1,42 @@
+{% extends "organizations_base.html" %}
+{% load i18n %}
+{% load org_tags %}
+{% block organization_switching_form %}
+	{% if request.user and request.user.is_authenticated %}
+	<form id="organizations-switch" action="{% url "switch-organizations" %}" method="post">{% csrf_token %}
+	    <select id="organizations-switch-select" name="organization" onchange="this.form.submit();"{% if organization_list|length == 1 and not perms.organizations.add_organization %} disabled="disabled"{% endif %}>
+	        {% for organization in organization_list %}<option value="{{ organization.id }}">{{ organization.name }}</option>{% endfor %}
+            <option value="-1" selected="selected">{% trans "Create a new Organization" %}</option>
+	    </select>
+	</form>
+	{%  endif %}
+{% endblock organization_switching_form %}
+
+{% block logframe_switching_form %}
+
+{% endblock logframe_switching_form %}
+{% block content %}
+<div class="organization-edit">
+   <h2>{% if form.instance.id %}{% blocktrans with logframe_name=form.instance.name %}Edit {{ logframe_name }} Organization{% endblocktrans %}{% else %}{% trans "Create a new Logframe" %}{% endif %}</h2>
+   <form method="post" class="pure-form pure-form-stacked" action=".">{% csrf_token %}
```

Comment:
We might want to agree on whether we're using British or US spelling of "Organisation"